tree/ntuple/inc/ROOT/RNTupleWriter.hxx

```diff
@@ -83,14 +118,33 @@ private:
                                                 const ROOT::RNTupleWriteOptions &options);
 
 public:
+   /// Creates an RNTupleWriter backed by `storage`, overwriting it if one with the same URI exists.
+   /// The format of the backing storage is determined by `storage`: in the simplest case it will be a local file, but
+   /// a different backend may be selected via the URI prefix. The "daos://" prefix will use the experimental DAOS
+   /// backend.
```

Comment:
I'm not sure if we want to mention DAOS here since it might only be available if enabled.